charts/px-central/templates/px-backup/pxcentral-alertmanager.yaml

```diff
@@ -164,7 +164,10 @@ stringData:
             <div
                 class="pb-16"
                 style="margin: 22px 0px;width: 400px; color: #bc1b06; font-size: 18px; font-weight: 500">
-                Critical Alert: Cluster Disconnected
+                Critical Alert: 
+                {{ if eq .Value 5.00 }}Cluster Disconnected
+                {{ else }}Cluster Offline
```

Comment:
please add the value condition for cluster offline (eq .Value 2.00) and an else case of unknown.